### PR TITLE
numbers in the restriction list entries are getting formatted

### LINKED ISSUE #1048: Format numbers in the restriction list entries
Numbers are not formatted in the restriction list entries.
This makes same very hard to read.

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #1048

### CODE REVIEW

File: components/ItemFilter/ItemFilterPropertiesDisplay.tsx [hunk 1]
```diff
@@ -69,6 +69,14 @@ function ItemFilterPropertiesDisplay(props: Props) {
                         return ''
                     }
 
+                    let greaterOrLessRegexp = new RegExp(/^[<>=]+/)
+                    if (!isNaN(Number(display.replace(greaterOrLessRegexp, '')))) {
+                        let symbols = display.match(greaterOrLessRegexp)
+                        let number = display.replace(greaterOrLessRegexp, '')
+                        let formattedNumber = numberWithThousandsSeperators(Number(number))
+                        display = symbols ? symbols[0] + formattedNumber : formattedNumber
```
Comment (Ekwav) [inline components/ItemFilter/ItemFilterPropertiesDisplay.tsx]: Doesn't this break with 1000-10000?

Comment (Ekwav) [inline components/ItemFilter/ItemFilterPropertiesDisplay.tsx]: Doesn't this break with 1000-10000?

**Comment by Ekwav** [comment]
Tests should pass

**Comment by matthias-luger** [comment]
The tests pass (at least locally), but somehow it seems to have problems fetching some archives when run on github. 
Gonna try to rerun it later

**Pull request merged**